### DISCUSSION TITLE
HM-ES-TX-WM / ESAx000 Support

### DISCUSSION THREAD

**Discussion by RealGue** [SukramJ/aiohomematic#708] [Q&A] [answered]
Hi,

da mir zum finalen Glück im HA nur noch der Verbrauch des Ferraris-Zählers fehlte, habe ich auf einem Pi4 Raspberrymatic mit nanocul aufgesetzt. Sensor ist ein "Energiespar-Sensor WZ" aus dem "Energiespar-Ampel"-Set. Also letztendlich die HM-ES-TX-WM mit Sensor.
Ich sehe im CUxD auch wunderbar die Pakete ankommen, ein Gerät konnte ich im Raspimatic aufbauen, das mir auch schön dort den Strom-Verbrauch im Power-Mode anzeigt.
Dann Homematic(IP) Local aus dem HACS installiert und meine Raspimatic angebunden.
Leider sehe ich da jetzt im HA nur nicht zugeordnete Entities ala "sensor.raspberrymatic_sv_svenergycounter_1042_cux2700001_1".
cux2700001_1 ist der Kanal, also wird zumindest schon was erkannt.

Habe ich einen Step vergessen? Oder wird das Device einfach nicht unterstützt?

Habt dank!
Jörn

**Comment by SukramJ** [comment, marked as answer]
Ist das ein CUxD Gerät?
CUxD wird nicht unterstützt, da für das von CUxD verwende BinRPC keine Python Implementierung existiert.

`sensor.raspberrymatic_sv_svenergycounter_1042_cux2700001_1` ist eine Systemvariable, und die werden natürlich unterstützt.

**Reply by RealGue** [reply]
Ok, danke, das war ja einfach. 
Dann versuche ich mal mein Glück, die Daten per MQTT zu HA zu shippen. 
Nochmals danke!

**Comment by emufan** [comment]
Hm, ich have das Gerät HM-ES-TX-WM auch (mit ES-Fer Sensor) und es ist wie gewünscht ohne eigenes Zutun im HA über die Integration.

![image](https://user-images.githubusercontent.com/22775515/202859176-08bdb5b1-92e5-4b1d-914b-3f80a9f332ed.png)

**Reply by SukramJ** [reply]
Aber nicht als CUxD Gerät.

**Reply by RealGue** [reply]
Korrekt. Ich habe ja nicht den normalen HM-ES-TX-WM, der sich als normales BidCos Gerät einbinden ließe, sondern das alte FS20-Teil. Die Pakete sind aber nahezu identisch, nur der Funk eben nicht.
Ich würde ja gerne den HM-ES-TX-WM kaufen, aber der scheint nicht mehr lieferbar zu sein.
Inzwischen hab ich Raspmatic wieder vom Pi geschmissen und liefere jetzt direkt beim HA ein.
